### PR TITLE
feat: Add Jira Server to Your Work

### LINKED ISSUE #9788: Jira Server: Enhanced Standups to be able to use Jira Server
## Issue - Enhancement

Today the "Your Work" drawer doesn't support Jira Server/Jira Data Center. This has been requested by our pals at P1.

### Acceptance Criteria (optional)
Users can:
   - See their Jira Server backlog tasks via "Your Work" in Standups

**Estimated effort:** X points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by Dschoordsch**
# Description

Fixes #9788 

## Demo

https://www.loom.com/share/0c3e669749d14780896e4185a8e74727?sid=96871a93-2b12-4b78-aa88-5ffcc24f987e

## Testing scenarios

- [ ] Integrate with Jira server org wide
- [ ] start a standup
- [ ] integrate with Jira server
- [ ] see your work

- [ ] create a new team in org
- [ ] see Jira server logo in your work drawer

- [ ] create unrelated org
- [ ] see no Jira server logo in you work drawer

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label https://github.com/ParabolInc/parabol/labels/Skip%20Maintainer%20Review if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by Dschoordsch** [comment]
@nickoferrall if testing the change is too much effort, it would be nice if you could confirm the other your work stuff is not broken.

File: packages/client/components/TeamPrompt/WorkDrawer/JiraServerIntegrationPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    return null
+  }
+
+  return (
+    <>
+      {isActive ? (
+        <JiraServerIntegrationResultsRoot teamId={teamMember.teamId} />
+      ) : (
+        <div className='-mt-14 flex h-full flex-col items-center justify-center gap-2'>
+          <div className='h-10 w-10'>{/* <img className='h-10 w-10' src={gitHubSVG} /> */}</div>
```
Comment (nickoferrall) [inline packages/client/components/TeamPrompt/WorkDrawer/JiraServerIntegrationPanel.tsx:74]: +1 can remove this

File: packages/client/components/TeamPrompt/WorkDrawer/JiraServerObjectCard.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      closeCopiedTooltip()
+    }, 2000)
+  }
+
+  const {summary, url, issueKey, projectName, updatedAt} = result
+
+  return (
+    <div className='rounded border border-solid border-slate-300 p-4 hover:border-slate-600'>
+      <div className='flex gap-2 text-xs text-slate-600'>
+        {/*<img src={issueIcon} />*/}
```
Comment (nickoferrall) [inline packages/client/components/TeamPrompt/WorkDrawer/JiraServerObjectCard.tsx:75]: +1 can remove this

File: packages/client/components/TeamPrompt/WorkDrawer/JiraServerIntegrationResults.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          <div className='-mt-14 flex h-full flex-col items-center justify-center'>
+            <img className='w-20' src={halloweenRetrospectiveTemplate} />
+            <div className='mt-7 w-2/3 text-center'>
+              {error?.message ? error.message : `Looks like you don’t have any issues to display.`}
+            </div>
+            <Link
+              to={`/team/${teamId}/integrations`}
+              className='mt-4 font-semibold text-sky-500 hover:text-sky-400'
+            >
+              Review your JiraServer configuration
```
Comment (nickoferrall) [inline packages/client/components/TeamPrompt/WorkDrawer/JiraServerIntegrationResults.tsx:100]: +1 might be an accident, but "Jira Server" should probably be two words as that's how it's written in the integration page

**Review by nickoferrall — APPROVED** [review]

**Pull request merged**